drivers/npmx/npmx_shell.c

```diff
@@ -4325,6 +4346,33 @@ static int cmd_reset(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+/* Creating subcommands (level 3 command) array for command "adc meas". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_adc_meas,
+			       SHELL_CMD(vbat, NULL, "Get battery voltage", cmd_adc_meas_vbat_get),
```

Comment:
"take" in this command has disappeared. Was that intentional?

Reply:
Yes